--- origsrc/vim74/src/mbyte.c	2014-01-09 23:24:09.686671600 -0600
+++ src/vim74/src/mbyte.c	2014-01-09 23:57:56.565140200 -0600
@@ -79,6 +79,10 @@
 
 #include "vim.h"
 
+#ifdef FEAT_CYGWIN_WIN32_CLIPBOARD
+#define WIN32UNIX
+#endif
+
 #ifdef WIN32UNIX
 # ifndef WIN32_LEAN_AND_MEAN
 #  define WIN32_LEAN_AND_MEAN
--- origsrc/vim74/src/os_unix.h	2013-12-08 03:12:15.918246100 -0600
+++ src/vim74/src/os_unix.h	2013-12-08 03:13:16.377713000 -0600
@@ -38,7 +38,6 @@
 #endif
 
 #if defined(__CYGWIN__) || defined(__CYGWIN32__)
-# define WIN32UNIX	/* Compiling for Win32 using Unix files. */
 # define BINARY_FILE_IO
 
 # define CASE_INSENSITIVE_FILENAME
